### DISCUSSION TITLE
It is possible to clear a Selection while in Crosshair Mode with Ctrl+D

### DISCUSSION THREAD

**Discussion by CaptainCooky** [Mojang/minecraft-editor#55] [Feedback]
While in Crosshair Mode, we are still able to clear the selection with the shortcut Ctrl+D. 
This could occur easily as users may often enter Crosshair Mode to navigate and inadvertently use the Ctrl key to "Sprint", as it's the default sprint key for Vanilla Minecraft. 
With WASD being the movement keys, it's easy to see how this could be annoying.

**Comment by minecraftchloe** [comment]
Interesting! We did this by design, however your feedback is making us question that decision.
